index.html

```diff
@@ -0,0 +1,156 @@
+               <li class="box-1">Workshop</li>
+               <p class="box-2">Join our webinars and live training sessions.</p>
+            </div>
+            <div class="box">
+               <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADMAAAAzCAYAAAA6oTAqAAAABmJLR0QA/wD/AP+gvaeTAAAGBklEQVRoge2aW4hXVRTGv+0tabSIFBVFc8xKHRMLxwlMKIgiauxBeorCIoQ0xV67qOgkUZbZS1ZEL0Vpd4oiIYSZ1DRExfE2aaWGpDOWt7Sc/PVw9thxnX3O2f+/Y/nQ93LY+3xr7bX2de2L9D8uTbhKBYBekqZLapQ0VlJ/SQcltUpaLekL59zJCD39JN0l6Q5J4yUNknRc0lZJn0r6yDn3V6X2RQMYD7RSjN+AF4CBOToG+v+/lehpBcZfLEduAo6XGJDGUeAJoKeX7+nTRyvQcQyY2N2O9AP2VWBEGmuBqcA3Vcr/CFweY2fUmAEWSJpvsn+Q9I6kw5KGSZomaXQFddSFNkkfStoraaCkhyXVGs4zzrlFVeg+HyTd42dTW5tJBnCa54ApQHNkjTcDUwLl9Qe2GO4BfHe9UGcaAoY0FvAd8CBwIseJE/5/bq8AGgNyk8ts7RHhzy0m3SHpszyycw5JV0iqyaHUSLrC8/LwmaR2k9dQYmeUM6NMutU5dzaPDDwkaXmJzuWeF4TXv73EjgxinBlk0rvziMDtkt5QdmLZYdJO0uvAbQXltpXYkUGMM71NOtg9gFGSVkrqZX5tlDTPf63eVV4uBNv61o4MbMEhnDHpy6Rz4chkSRMk1SkJS64OyE+S9GWO7qslrQFWS9omaYukb51zJyT1MdzOMkNjnPnVpKcA6yXdHClfhmGSZqTSncAmZSvG2pFBjDGHTbpW2UWtO9FLUn0g385u8QDGAu8AnXFr4EVHJ4k9Yypxoh/wInCmykKXAfXAGGAYcKXXe6VPj/H/l1Wp/wywFMhbx845UgfsiFC4Fng1kL8VKJ11fFl9PN/iVa+/DNuBcXnKG4AjEUoAPgc2mbxOoD6l7xpgCUm03Aa0+PSIFKeebDfe5PXH4AjQYB2pBToC5LPAu8DLgXyLpSl9jwOncww4DcxOcZfmlJvGK8B7OeV2ALVdyhzhvcZP+KiWZDIowkF8H/aOxGCW59d4+SKM9dxbCe+rWgAnYFrg50ZgsGm9toLC5qS6Vl6LWJwChnu5uQW8NmPHYG+fRaOAD0xmBzA0MKYW5hS2H+jrOUsiHelCk5fr6/WEsDBgy1Cyw2KVAkqezJkgRhHuszNTnEq3xi0p2ZmB/2fJid2Apwx3n8h2i6KN1y7D/YPUfB/4X4adKdkary+NXQW22A3c6R7KhivB4x3gWmX3+H0kPZBKVxpypMt+QNngcjRdM1UWE0z6kID3jYd5Y2ZRTu2mx8yzFbbMYi9XzZixa+LKotlsiFGwp8CortlsBMksFYP0bDangLfb2DEE+C7Aa+xaZ1oCP/cBU72CuhLD0uvM7DIvPB7z/Jh1ZoznTiXcgs10HZAAIwlHAJCsvK+YvLIIYBb5LXQK74jnxkQAy4GVOfragZG2H3Z3bDYcaCKptd3+24TvWp7THbFZ+AgKGEcSjZZhHbAikF9J1NybC4uaW/FhTlEhNSRNf6nuZ/4EnqdsP2OcugF4m0trp/k2cEO0EwGnmv5jJ7rQVGZrzLmZvTTaI2mDIo5+qkSn17/H5A8oE4w5nbnKpJudczNIzs0aJN2of87NhkXoszig5PrwvHMz4C2dfyRr7agcwMemuVfk8EaRzPkWG4A7/deinfyo+DXD/bDM1phuZk80g1cRzrk9ku4P8CdJesl/rd7pXi7GNqu3VCCEX0z6ujyic+5rSY8qex5tz7rOSnrEObemoFwboVs7KgcwzzT3YaCwEiiPz2aVyDtfThpzy2yNaZl1Jj1A0j1Fhkg6JinvLcBJSccpuDmTdK+ys9f6EjvLQXKnecDU0magv+E5ktOTUAQeQrPnO6MndKe5n4g7zdjb5vmSFpjsvZLeVFLTQyXdJ+naGH0G30v6RMkUXaPwbfPTzrnFVejOgiReq/YdwDck+5CY4DGE6HcAlTg0keTFRCx+JTkP6+Hlq3mhcRSwe/1ucyjm7cwR4DkgGH4Q/3ZmG1BXiX3VvGrqqWR8TJN0vZIw45CSV01fKXnV9HuEnhpJdysJg+qUXMC2S9qpZAx9clFfNf2PfxF/AxZA0JFEO8bZAAAAAElFTkSuQmCC">
+               <li class="box-1">DC Ignite</li>
+               <p class="box-2">Join our webinars and live training sessions.</p>
+            </div>
+          </div>
+        </ul>
+        <a class="mp-1">SEE THE WHOLE PROGRAM</a>
+        <button class="button">Join Interior Symposium 2022</button>
+     </div>
+    </section>
```

Comment:
- [ ] Kindly  adjust Layout: composition and space between elements for section. 

Your Main program section Look Likes this : 
![C2](https://user-images.githubusercontent.com/93598189/191450394-99b54139-47d4-4080-b813-86c052e2a3d9.PNG)

The Design Main section Look Likes this : 
![C1](https://user-images.githubusercontent.com/93598189/191450501-e71baeb4-336f-4ee4-9a11-a5fca5d7c4d1.PNG)

Reply:
- [ ] This still needs work to be done. 

![image](https://user-images.githubusercontent.com/58504655/191569999-16b03a10-becc-4cd1-befa-26c6526ee9df.png)